server/test/write_authzmodel.go

```diff
@@ -111,48 +42,6 @@ func WriteAuthorizationModelTest(t *testing.T, datastore storage.OpenFGADatastor
 			},
 			err: serverErrors.ExceededEntityLimit("type definitions in an authorization model", datastore.MaxTypesPerAuthorizationModel()),
 		},
-		{
-			name: "succeeds_with_empty_relations",
-			request: &openfgapb.WriteAuthorizationModelRequest{
-				TypeDefinitions: []*openfgapb.TypeDefinition{
-					{
-						Type: "repo",
-					},
-				},
-			},
-		},
-		{
-			name: "succeeds_with_zero_length_relations",
-			request: &openfgapb.WriteAuthorizationModelRequest{
-				TypeDefinitions: []*openfgapb.TypeDefinition{
-					{
-						Type:      "repo",
-						Relations: map[string]*openfgapb.Userset{},
-					},
-				},
-			},
-		},
-		{
-			name: "fails_if_the_same_type_appears_twice",
-			request: &openfgapb.WriteAuthorizationModelRequest{
-				StoreId: storeID,
-				TypeDefinitions: []*openfgapb.TypeDefinition{
-					{
-						Type: "repo",
-						Relations: map[string]*openfgapb.Userset{
-							"admin": {Userset: &openfgapb.Userset_This{}},
-						},
-					},
-					{
-						Type: "repo",
-						Relations: map[string]*openfgapb.Userset{
-							"admin": {Userset: &openfgapb.Userset_This{}},
-						},
-					},
-				},
-			},
-			err: serverErrors.InvalidAuthorizationModelInput(typesystem.ErrDuplicateTypes),
-		},
 		{
 			name: "fails_if_a_relation_is_not_defined",
```

Comment:
What's the rationale for keeping some and removing others? Can we move all of them into the new test structure instead?